core/metacling/src/TCling.h

```diff
@@ -202,6 +202,7 @@ class TCling : public TInterpreter {
    virtual void Initialize();
    void    InspectMembers(TMemberInspector&, const void* obj, const TClass* cl, Bool_t isTransient);
    Bool_t  IsLoaded(const char* filename) const;
+   Bool_t  IsLibraryLoaded(const char* libname) const;
```

Comment:
This should be added to the base class, `TInterpreter`, as a virtual interface (like all the other functions are).

Reply:
Agreed about FindDynamicLibrary.

https://github.com/root-project/root/pull/2128/files#diff-f77c920d2b220f3dd0edb678d4815facR155
Is this not enough?